reporters_db/data/reporters.json

```diff
@@ -2358,6 +2400,20 @@
             }
         }
     ],
+    "Blatchf. & H.": [
+        {
+            "cite_type": "scotus_early",
```

Comment:
is it Scots early... there is another issue about these ... that someone resolved... that I need to review.  can you just check into it.